gce_ovf_import_tests/test_config/project_config.go

```diff
@@ -0,0 +1,35 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package testconfig
+
+// Project is details of test Project.
+type Project struct {
+	TestProjectID        string
+	ServiceAccountEmail  string
+	TestZone             string
+	ServiceAccountScopes []string
+}
+
+// GetProject creates a Project object for given project ID and zone
+func GetProject(projectID, testZone string) *Project {
+	return &Project{
+		TestProjectID:       projectID,
+		TestZone:            testZone,
+		ServiceAccountEmail: "default",
+		ServiceAccountScopes: []string{
+			"https://www.googleapis.com/auth/cloud-platform",
```

Comment:
This is a very broad scoping, is this needed? Can we get away with a smaller subset?

Reply:
This configuration is based on OS Config e2e tests which use the same service account scopes.

Public docs state the best practice is to grant 'cloud-platform' scope to an instance then limit it's permissions using IAM: https://cloud.google.com/compute/docs/access/service-accounts#accesscopesiam

Since this is internal code running e2e tests in a test project, what are your concerns in having broad set of permissions on a particular instance?